dprint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "markdown": {
+  },
+  "dockerfile": {
+  },
+  "excludes": [
+    "deprecated/*"
+  ],
+  "plugins": [
+    "https://plugins.dprint.dev/markdown-0.17.0.wasm",
```

Comment:
How do we keep these updated? O don't think dependable support this.

Reply:
Currently there isn't any automated way, but apparently warning about outdated plugin versions is on the roadmap.